src/transformState.js

```diff
@@ -46,7 +46,25 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const obj of transforms) {
+    switch (obj.operation) {
+      case 'addProperties':
+        Object.assign(state, obj.properties);
+        break;
+      case 'removeProperties':
```

Comment:
add a new line before new case

Reply:
Why I needed default case? it will have nothing